.github/workflows/ci_test-full.yml

```diff
@@ -106,60 +113,81 @@ jobs:
+        pip uninstall horovod
+        pip list
       shell: bash
 
-    - name: Reinstall Horovod if necessary
-      if: runner.os != 'windows'
-      env:
-        HOROVOD_BUILD_ARCH_FLAGS: "-mfma"
-        HOROVOD_WITHOUT_MXNET: 1
-        HOROVOD_WITHOUT_TENSORFLOW: 1
-      run: |
-        HOROVOD_BUILT=$(python -c "import horovod.torch; horovod.torch.nccl_built(); print('SUCCESS')" || true)
-        if [[ $HOROVOD_BUILT != "SUCCESS" ]]; then
-          pip uninstall -y horovod
-          echo $(grep "horovod" requirements/extra.txt) > requirements/horovod.txt
-          pip install --no-cache-dir -r requirements/horovod.txt
-        fi
-        horovodrun --check-build
-      shell: bash
+    # - name: Reinstall Horovod if necessary
+    #   if: runner.os != 'windows'
+    #   env:
+    #     HOROVOD_BUILD_ARCH_FLAGS: "-mfma"
+    #     HOROVOD_WITHOUT_MXNET: 1
+    #     HOROVOD_WITHOUT_TENSORFLOW: 1
+    #   run: |
+    #     HOROVOD_BUILT=$(python -c "import horovod.torch; horovod.torch.nccl_built(); print('SUCCESS')" || true)
+    #     if [[ $HOROVOD_BUILT != "SUCCESS" ]]; then
+    #       pip uninstall -y horovod
+    #       echo $(grep "horovod" requirements/extra.txt) > requirements/horovod.txt
+    #       pip install --no-cache-dir -r requirements/horovod.txt
+    #     fi
+    #     horovodrun --check-build
+    #   shell: bash
 
     - name: Cache datasets
       uses: actions/cache@v2
       with:
         path: Datasets
         key: pl-dataset
 
-    - name: Tests
+    - name: Tests - ./pytorch_lightning/
       run: |
-        # NOTE: do not include coverage report here, see: https://github.com/nedbat/coveragepy/issues/1003
-        coverage run --source pytorch_lightning -m pytest pytorch_lightning tests -v --durations=50 --junitxml=junit/test-results-${{ runner.os }}-py${{ matrix.python-version }}-${{ matrix.requires }}-${{ matrix.release }}.xml
+        python -m pytest pytorch_lightning -v
 
-    - name: Examples
+    - name: Tests - ./tests/
       run: |
-        python -m pytest pl_examples -v --durations=10
-
-    - name: Upload pytest results
-      uses: actions/upload-artifact@v2
-      with:
-        name: pytest-results-${{ runner.os }}-${{ matrix.python-version }}-${{ matrix.requires }}-${{ matrix.release }}
-        path: junit/test-results-${{ runner.os }}-${{ matrix.python-version }}-${{ matrix.requires }}-${{ matrix.release }}.xml
-      if: failure()
-
-    - name: Statistics
-      if: success()
-      run: |
-        coverage report
-        coverage xml
-
-    - name: Upload coverage to Codecov
-      uses: codecov/codecov-action@v1
-      if: always()
-      # see: https://github.com/actions/toolkit/issues/399
-      continue-on-error: true
-      with:
-        token: ${{ secrets.CODECOV_TOKEN }}
-        file: coverage.xml
-        flags: cpu,pytest,python${{ matrix.python-version }}
-        name: CPU-coverage
-        fail_ci_if_error: false
+        python -m pytest tests/accelerators -v
```

Comment:
Interesting, so just accelerators still segfaults.

What if you disable accelerators and just test callbacks? (for example)